src/domains/transaction/pages/SendTransfer/SendTransfer.tsx

```diff
@@ -157,15 +157,27 @@ export const SendTransfer = () => {
 	const submitForm = async () => {
 		clearErrors("mnemonic");
 
-		const { fee, mnemonic, secondMnemonic, recipients, senderAddress, smartbridge } = getValues();
+		const {
+			fee,
+			mnemonic,
+			secondMnemonic,
+			recipients,
+			senderAddress,
+			smartbridge,
+			encryptionPassword,
+		} = getValues();
 
 		const isMultiPayment = recipients.length > 1;
 
 		const transactionType = isMultiPayment ? "multiPayment" : "transfer";
+
```

Comment:
@faustbrian generating wif and passing it to TransactionInputs in cases where encryption password is set. Mnemonic comes undefined when wif is available. This is used in all transaction forms. Do you see any issues?